League/Repository/ClientRepository.php

```diff
@@ -86,6 +86,6 @@ private function isGrantSupported(ClientModel $client, ?string $grant): bool
             return true;
         }
 
-        return \in_array($grant, $client->getGrants());
+        return \in_array($grant, $client->getGrants(), true);
```

Comment:
`$grant` contains string,
`$client->getGrants()` returns array of `Trikoder\Bundle\OAuth2Bundle\Model\Client`
So we can not use `strict = true` here

Reply:
oh okay thank you for pointing this out. will fix.